BraveRewardsUI/Publisher/PublisherSummaryView.swift

```diff
@@ -51,7 +42,16 @@ class PublisherSummaryView: UIView {
   
   public override init(frame: CGRect) {
     super.init(frame: frame)
-    
+  }
+  
+  convenience init(emptyView: Bool) {
+    self.init()
+    if !emptyView {
```

Comment:
So the fact that we do this just makes me think this logic should be split out and set as a whole new view in `WalletViewController` same as we do here:

https://github.com/brave/brave-rewards-ios/blob/ef5ac14d32230dea09f6b6aa5122ae875a6dd8a1/BraveRewardsUI/Wallet/WalletViewController.swift#L205-L216

Basically just adding a check for local within the `isEnabled` flow and setting `contentView` to a `RewardsSummaryView` 

This would let us take away all these odd changes for handling layout and such in the publisher view stuff since its separate anyways and clean things up significantly here. Should make layout logic easier too

Reply:
Sounds good! Will add